### PR TITLE
[INFINITY-1366] Change Elastic service user to 'nobody'

### PR DESCRIPTION
**Pull request by NimaVaziri**

### CODE REVIEW

File: frameworks/elastic/universe/config.json [hunk 1]
```diff
@@ -34,6 +34,11 @@
           "description": "Elasticsearch deploy strategy. [parallel, serial]",
           "type": "string",
           "default": "parallel"
+        },
+        "mesos_api_version": {
```
Comment (nickbp) [inline frameworks/elastic/universe/config.json]: Is anything using this value? Not mentioned elsewhere in the PR.

Comment (NimaVaziri) [inline frameworks/elastic/universe/config.json]: It's apparently needed to make strict work: https://github.com/mesosphere/dcos-commons/pull/1013/files#diff-a4edd5b45d2ef4fda8fd5a8e90348736R115

Comment (nickbp) [inline frameworks/elastic/universe/config.json]: Looks like there's supposed to be a `"MESOS_API_VERSION": "{{service.mesos_api_version}}"` in marathon.json. See examples in kafka, helloworld, cassandra.

Comment (NimaVaziri) [inline frameworks/elastic/universe/config.json]: Yeah, will patch.

Comment (nickbp) [inline frameworks/elastic/universe/config.json]: OK per conversation elsewhere this option will likely be removed from everything except hello-world soon. Can just add it here for now as a no-op for CI compatibility, and then remove it again in a later pass.

Comment (NimaVaziri) [inline frameworks/elastic/universe/config.json]: Dropped patch.

**Pull request merged**